bin/smack.py

```diff
@@ -15,13 +15,71 @@
 import sys
 import tempfile
 from threading import Timer
+from toSVCOMPformat import *
+from token_replace import *
 
 VERSION = '1.5.1'
 temporary_files = []
 
+def rewriteIExtensionToC(args):
```

Comment:
We thought about this, but then apparently forgot to look in to this.

I'll take a look now.